cupy/cuda/common.pxd

```diff
@@ -1,5 +1,7 @@
 # distutils: language = c++
 
+from libcpp cimport bool as bool_t
```

Comment:
Nit: I am not sure if `bool_t` is used in other places of the code base or `bint` is favored over it, but we should stick to one convention if possible.

Reply:
Yeah I know I said `bint` is favored in Cython 😅, but this is for template specialization for C++ `bool`. I am not sure if `bint` is always translated to `bool`, so tried to be on the safe side. Perhaps I could rename it to `cpp_bool` for style consistency? See
https://github.com/cupy/cupy/blob/0c841101dbbc0afc59f7dd340d8ba66a39795146/cupy/core/internal.pxd#L1